mds/management/commands/poll_providers.py

```diff
@@ -415,6 +417,7 @@ def create_device(status_change):
         "model": "",
         "propulsion": status_change["propulsion_type"],
         "manufacturer": "",
+        "dn_status": enums.DEVICE_STATUS.unknown.name,
```

Comment:
Remove these changed to the poller, you are reverting a later commit.

Reply:
damn, good point